packages/wonder-blocks-modal/components/modal-header.js

```diff
@@ -11,34 +11,48 @@ import {
     LabelMedium,
 } from "@khanacademy/wonder-blocks-typography";
 
-type Props = {|
+type Common = {|
     /**
      * The main title rendered in larger bold text.
      */
     title: string,
 
     /**
-     * The dialog subtitle.
+     * Whether to display the "light" version of this component instead, for
+     * use when the item is used on a dark background.
      */
-    subtitle?: string,
+    light: boolean,
 
     /**
-     * Adds a breadcrumb-trail, appearing in the ModalHeader, above the title.
+     * An id to provide a selector for the title element.
      */
-    breadcrumbs?: React.Element<typeof Breadcrumbs>,
+    titleId: string,
+
+    // Keep flow happy
+    subtitle?: void,
+    breadcrumbs?: void,
```

Comment:
Without these, flow complains about `subtitle` and `breadcrumbs` not being available on props at all b/c these are exact object types, flow looks for `subtitle` in each of `Common`, `WithSubtitle` and `WithBreadcrumbs`.  I also tried making the types inexact and flow still complains when destructuring which is a little odd.

Reply:
It might be worth putting this exact comment inline here in the code.